pandas/core/generic.py

```diff
@@ -1591,62 +1591,87 @@ def _repr_latex_(self):
     # I/O Methods
 
     _shared_docs['to_excel'] = """
-    Write %(klass)s to an excel sheet
-    %(versionadded_to_excel)s
+    Write %(klass)s to an excel sheet.
+
+    To write a %(klass)s to an excel .xlsx file it is necessary to first create
+    an ExcelWriter object with a target file name, and specify a sheet in the
```

Comment:
Since ExcelWriter is a class I believe it should be between backticks to make it itallic like
```
`ExcelWriter`
```

Reply:
ok